docs/src/tutorials/yocto-linux.md

```diff
@@ -0,0 +1,102 @@
+# Build Thin Edge for a Yocto Linux distribution
+
+Yocto Project enables you to create a customised Linux distribution for your IoT devices. You can select the base image
+and add layers, containing software that you need on your image. In this tutorial, we will add Thin Edge using the
+`meta-tedge` layer. For more information, see the [getting started document on Yocto Project
+website](https://www.yoctoproject.org/software-overview/).
+
+
+`meta-tedge` is supported for **Yocto version 4.0 "Kirkstone"**. Additionally, we will depend `meta-openembedded` layer
+for Mosquitto.
+
+## Installation
+
+If you are not familiar with building Yocto distribution or you have not configured your build host yet, we strongly
+recommend to look into [official yocto documentation](https://docs.yoctoproject.org/brief-yoctoprojectqs/index.html)
+as the installation process will now skip all information that were mentioned there! For workspace organization or
+raspberry pi distribution, we also recommend this [guide](https://github.com/jynik/ready-set-yocto)
+
+> Most of the installation process is based on [Yocto Project Quick Build
+guide](https://docs.yoctoproject.org/brief-yoctoprojectqs/index.html).
+
+Before starting, be sure your hosts meets the following requirements:
```

Comment:
I would be more explicit, highlighting this is about the host used to build the image.

```suggestion
Before starting, be sure the host, where you plan to build a new Yocto image with thin-edge, meets the following requirements:
```